mapo/envs/navigation.py

```diff
@@ -92,13 +92,25 @@ def _transition_fn(self, state, action):
         if self._deceleration_zones:
             deceleration = self._deceleration()
 
-        noise = self._sample_noise()
-        next_state = state + (deceleration * action) + noise
-        return next_state
+        position = state + (deceleration * action)
+        dist, next_state = self._sample_noise(position)
+        log_prob = dist.log_prob(tf.stop_gradient(next_state))
+        return next_state, log_prob
+
+    def _transition_log_prob_fn(self, state, action, next_state):
```

Comment:
There you go... I called this method `_transition_log_prob_fn` to be consistent with the other TF functions (e.g., `_transition_fn` and `_reward_fn`)